packages/testutil/testchain/test_chain_ledger.go

```diff
@@ -57,15 +58,15 @@ func (tcl *TestChainLedger) ChainID() isc.ChainID {
 	return tcl.chainID
 }
 
-func (tcl *TestChainLedger) MakeTxChainOrigin(committeeAddress *cryptolib.Address) *isc.StateAnchor {
+func (tcl *TestChainLedger) MakeTxChainOrigin(committeeAddress *cryptolib.Address) (*isc.StateAnchor, uint64) {
```

Comment:
should be `coin.Value` if it's a coin balance

```suggestion
func (tcl *TestChainLedger) MakeTxChainOrigin(committeeAddress *cryptolib.Address) (*isc.StateAnchor, coin.Value) {
```